mcs/class/corlib/ReferenceSources/RuntimeType.cs

```diff
@@ -32,7 +32,7 @@
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
 using System.Globalization;
-#if MONO_COM
+#if FEATURE_COMINTEROP
```

Comment:
I think we are trying to make sure some MONO symbol wraps any change we have made to the ReferenceSources/ directory. So maybe this should be #if MONO && FEATURE_COMINTEROP